smart-events-user-dashboard/src/Login.tsx

```diff
@@ -1,34 +1,54 @@
 import * as React from 'react';
+import './App.css'
 import {
-  Container, 
+  Box,
```

Comment:
I've never seen this container before. Awesome stuff, I love when people iterate on our existing code practices and find new innovative ways of doing it.